pkg/cmd/update-tls-artifacts/metadata/requirements.go

```diff
@@ -0,0 +1,34 @@
+package metadata
+
+import (
+	"github.com/openshift/library-go/pkg/certs/cert-inspection/certgraphapi"
+)
+
+type secretCompareFunc func(actual, expected certgraphapi.PKIRegistryCertKeyPairInfo) error
+type configMapCompareFunc func(actual, expected certgraphapi.PKIRegistryCertificateAuthorityInfo) error
+
+type Requirement struct {
+	Name                 string
+	SecretCompareFunc    secretCompareFunc
```

Comment:
where is this used?

Reply:
Rebase ate my commit which used this in test. This allows us to customize diff function (not sure if it would be useful later on though)